keyvi/include/keyvi/dictionary/dictionary_merger.h

```diff
@@ -99,7 +121,8 @@ class DictionaryMerger final {
    *
    * @params params merger parameters
    */
-  explicit DictionaryMerger(const merger_param_t& params = merger_param_t()) : dicts_to_merge_(), params_(params) {
+  explicit DictionaryMerger(const keyvi::util::parameters_t& params = keyvi::util::parameters_t())
```

Comment:
Probably it worth to add `using keyvi::util::parameters_t` on class/function level and then just use `parameters_t` as a type in order to have less dense code.